docs/guide/ecosystem/documentation.md

```diff
@@ -151,38 +155,31 @@ props: {
 
 #### Pros
 
-- Displays a live editor to try components
-- Meant for design system creation (powers vue-design-system)
-- A strong React community
+- Displays a live editor to try components even on the static website
+- Meant for design system creation and showcase (powers vue-design-system)
+- Multiple components examples can be shown in the same markdown page
 
```

Comment:
I want to stress again, please mention it automagically extracts data from the component, like props, slots, events etc. This is a major PRO when it comes to pure documentation generators.

Reply:
Oh yeah I forgot the main selling point.